pagefaultgames/pokerogue
# [QoL] Enable female gender for all default starters and all existing s…

## What are the changes?
Small change to allow gender changes in default starter pokemon 

## Why am I doing these changes?
some people want notboy pokemon
Closes #2103 

## What did change?
-set option for default starter pokemon gender 
-add migration for above

## How to test the changes?
start a new save, verify female exists for all 27 starters, then in existing save verify addition

## Checklist
- [y] There is no overlap with another PR?
- [y] The PR is self-contained and cannot be split into smaller PRs?
- [y] Have I provided a clear explanation of the changes?
- [y] Have I tested the changes (manually)?
    - [y?] Are all unit tests still passing? (`npm run test`) 6 errors on main
- [n] Are the changes visual?
  - [n] Have I provided screenshots/videos of the changes?
